server/index.js

```diff
@@ -39,7 +32,7 @@ if (process.env.LIBRATO_TOKEN && process.env.LIBRATO_USER) {
   });
   librato.start();
 
-  options.onMetric = function onMetricProduction(metric = "", value = 1, ctx = {}) {
+  onMetric = function onMetricProduction(metric = "", value = 1, ctx = {}) {
```

Comment:
We should also remove librato integration and use `req.hull.metric` (https://hull.gitbooks.io/docs/content/v/beta2/context.html)